pkg/transport/filtering/object_filtering_map.go

```diff
@@ -43,9 +46,13 @@ type ObjectFilteringMap struct {
 }
 
 func (filteringMap *ObjectFilteringMap) CleanObjectSpecifics(object *unstructured.Unstructured) {
-	filteringFunction, found := filteringMap.gvkToFilteringFunc[object.GetObjectKind().GroupVersionKind()]
+	gvk := object.GetObjectKind().GroupVersionKind()
+	filteringFunction, found := filteringMap.gvkToFilteringFunc[gvk]
 	if !found {
+		klog.V(4).InfoS("No specifics", "gvk", gvk)
 		return // if no filtering function was defined for this gvk, do not clean any field
+	} else {
+		klog.V(4).InfoS("Found specifics", "gvk", gvk)
 	}
```

Comment:
can we remove the log from this function?
this will thrash the controller log on every object that is being wrapped, make it harder to follow the log.

Reply:
That was an oversight. Fixed.